models/labels/addresses/infrastructure/token_standard/labels_token_standards_arbitrum.sql

```diff
@@ -10,14 +10,16 @@ SELECT distinct 'arbitrum' AS blockchain
 , 'infrastructure' AS category
 , 'hildobby' AS contributor
 , 'query' AS source
-, date('2023-03-02') AS created_at
-, NOW() AS modified_at
+, date '2023-03-02' AS created_at
+, NOW() AS updated_at
```

Comment:
Why do we rename the column name?

Reply:
There is a typo in the original spell.
So all labels spells should have `updated_at` field, and ideally when in the roll-up spell this may help them to be consistent.
There are some labels having typo in this field with name `modified_at`, and the roll-up was `select * from ...`, so I fixed them along the way.